t4d

```diff
@@ -83,7 +83,7 @@ case $1 in
         else
             source $Tools4Dev_PATH/tools4Dev.env &> /dev/null
         fi
-        t4dLoadProjectConf &> /dev/null
+        t4dLoadProjectConf &> /dev/null || t4dLoadProjectConf
```

Comment:
"if it breaks, try again" ?

Reply:
nop :p 

"if it breaks, at least show me some log this time"

Reply:
Ah